Sources/PointFree/Rss.swift

```diff
@@ -0,0 +1,273 @@
+import Foundation
+import Html
+import Optics
+import Prelude
+
+struct RssChannel {
```

Comment:
this file contains a bunch of models for RSS things as well as how to turn those models into `Node`s.